metricflow/plan_conversion/time_spine.py

```
@@ -1,109 +1,180 @@
+
+    start = _timestamp_text(start_time)
+    end = _timestamp_text(end_time)
+
+    if sql_engine in (SqlEngine.POSTGRES, SqlEngine.GREENPLUM, SqlEngine.DUCKDB):
+        return f"""\
+SELECT {column_name}
+FROM GENERATE_SERIES(
+  CAST('{start}' AS TIMESTAMP),
+  CAST('{end}' AS TIMESTAMP),
+  INTERVAL '1 day'
+) AS time_spine({column_name})"""
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)